config/jobs/kubernetes/publishing-bot/publishing-bot-presubmits.yaml

```diff
@@ -47,6 +47,27 @@ presubmits:
     annotations:
       testgrid-dashboards: sig-release-publishing-bot
       testgrid-tab-name: validate-rules
+  - name: pull-publishing-bot-test-kubernetes-master
+    always_run: true
```

Comment:
do we want to do `always_run: false` so we can test this for a bit of time before we run it always?

Reply:
Done.